### PR TITLE
Migrate 603-spring-web-smallrye-openapi from BeefyTS

### PR DESCRIPTION
**Pull request by jsmrcka**
Closes https://github.com/quarkus-qe/quarkus-test-suite/issues/98

### CODE REVIEW

File: spring/spring-web/pom.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>spring-web</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: spring-web</name>
```
Comment (Sgitario) [inline spring/spring-web/pom.xml:12]:
```suggestion
    <name>Quarkus QE TS: Spring: Spring Web</name>
```

**Review by Sgitario — CHANGES_REQUESTED** [review]
The windows action is failing because this module is using testcontainers which is not supported in Windows.
You need to add this profile in the pom.xml of this new module: https://github.com/quarkus-qe/quarkus-test-suite/blob/909dfb5188baf4deb823a25b9a8dc738ed0be683/sql-db/panache-flyway/pom.xml#L43

File: spring/spring-web/pom.xml (hunk 1 above)
Comment (jsmrcka) [reply spring/spring-web/pom.xml]: Thanks for pointing it out. I also fixed the name for previously migrated `spring/spring-data` module and added it to this PR.

**Review by Sgitario — APPROVED** [review]

**Comment by Sgitario** [comment]
run tests

**Pull request merged**